backend/src/map_model.rs

```diff
@@ -1024,6 +1141,7 @@ pub enum Command {
     SetModalFilter(RoadID, Option<ModalFilter>),
     SetDiagonalFilter(IntersectionID, Option<DiagonalFilter>),
     SetTravelFlow(RoadID, TravelFlow),
+    SetTurnRestrictions(IntersectionID, Vec<(RoadID, RoadID)>),
```

Comment:
Every time I design savefile formats or think through how to implement the command pattern for undo, I hit this question about objects with lists. We could also have a command for "add" and "delete" one individual TR. In our case, the list order here doesn't matter.

I started with this because it was simpler and the choice doesn't seem to matter here, but open to opinions